javascript/javascript1/week2/Candy-helper.js

```diff
@@ -0,0 +1,53 @@
+const candyTable = {
+    sweet: 0.5,
+    chocolate: 0.7,
+    toffee: 1.1,
+    chewingGum: 0.03,
+}
+let boughtCandyPrices = [];
+const yourLimit = Math.floor(Math.random() * 100);
+let youBalance = 0;
+
+function canBuyMoreCandy() {
+    for (i = 0; i < boughtCandyPrices.length; i++) {
+        youBalance += boughtCandyPrices[i]
+    }
+
+    if (youBalance < yourLimit) {
+        return true
+    } else {
+        return false
+    }
+
+}
+
+function addCandy(candyType, weight) {
+    if (canBuyMoreCandy) {
```

Comment:
if we have multiple levels of if, else condition, better to go for switch condition

Reply:
yah i will rewrite it again